specification/databoxedge/resource-manager/readme.md

```diff
@@ -28,6 +28,14 @@ These are the global settings for the DataBox Edge API.
 openapi-type: arm
 tag: package-2019-08
```

Comment:
Do you need to update the default tag? Otherwise, when generating SDK, by default, it would  use the tag of package-2019-08